src/main/java/com/ncov/module/controller/request/material/MaterialRequest.java

```diff
@@ -14,16 +14,20 @@
 
     @ApiModelProperty(value = "物资信息")
     private List<MaterialDto> materials;
-    @ApiModelProperty(value = "发布/供货机构名称")
+    @ApiModelProperty(value = "发布/供货机构名称", required = true)
     private String organisationName;
-    @ApiModelProperty(value = "收货/发货地址")
+    @ApiModelProperty(value = "收货/发货地址", required = true)
     private String address;
-    @ApiModelProperty(value = "联系人")
+    @ApiModelProperty(value = "联系人", required = true)
     private String contactorName;
-    @ApiModelProperty(value = "联系电话")
+    @ApiModelProperty(value = "联系电话", required = true)
     private String contactorPhone;
     @ApiModelProperty(value = "物资图片, 支持多张")
     private List<String> imageUrls;
     @ApiModelProperty(value = "备注")
     private String comment;
+    @ApiModelProperty(value = "第几页", notes = "只有分页查询才需要传入数据", required = false)
```

Comment:
Not sure why we are putting pagination parameters in the request body?

Reply:
forget delete